pkg/gui/controllers/branches_controller.go

```diff
@@ -432,11 +432,19 @@ func (self *BranchesController) createPullRequestMenu(selectedBranch *models.Bra
 			{
 				LabelColumns: fromToLabelColumns(branch.Name, self.c.Tr.LcSelectBranch),
 				OnPress: func() error {
+					if len(self.model.Remotes) == 1 {
+						toRemote := self.model.Remotes[0].Name
+						self.c.Log.Debugf("PR will target the only existing remote '%s'", toRemote)
+						return self.promptForTargetBranchNameAndCreatePullRequest(branch.Name, toRemote)
+					}
+
 					return self.c.Prompt(types.PromptOpts{
-						Title:               branch.Name + " →",
-						FindSuggestionsFunc: self.helpers.Suggestions.GetBranchNameSuggestionsFunc(),
-						HandleConfirm: func(targetBranchName string) error {
-							return self.createPullRequest(branch.Name, targetBranchName)
+						Title:               "Select Target Remote",
```

Comment:
I'd make this translatable when finalizing the PR.

Reply:
When you do, be sure to use `Sentence case`